build.sbt

```diff
@@ -132,6 +142,7 @@ lazy val lintFlags = forScalaVersions {
       "-Xlint:_,-unused",
       "-Xfatal-warnings",
       "-Yno-adapted-args",
+      "-Yrangepos",
```

Comment:
Why did you enable this?

Reply:
It's required for the SemanticDB compiler plugin (https://scalacenter.github.io/scalafix/docs/users/installation.html). I added a comment to make it explicit.